src/README.md

```diff
@@ -24,6 +26,27 @@ cd reason
 opam pin add -y reason .
 ```
 
+### With esy
+
+The esy workflow is still experimental.
+If you would like to help test, you can try it with the following commands.
+If you already use OPAM for Reason, take care to deactivate OPAM first
+or ensure your node binaries precede OPAM's in your path.
```

Comment:
I think it’s the opposite. You’d want node binaries to come after opam’s. Also, esy does a good job of scrubbing the env during builds so I’m not sure this warning is necessary. Did you run into an issue?

Reply:
I think this actually may not be necessary, although so far I found that esy basically eliminated my need to use opam directly. Probably best to remove this warning.

Reply:
But your initial concern about people having opam already installed is well-founded - it happens a lot. I think we should be okay without this warning.

Reply:
Hm... I think if you use `esy ...` prefix (`esy x make test` and so on) in workflow the fact that OPAM is installed and how it's configured is totally irrelevant. It's a bug in Esy in any other case.